.ci/run_tests.sh

```diff
@@ -104,8 +104,8 @@ else
     ${VOLUMES} \
     --env-file .docker/docker-variables.env \
     --env PUSH_TO_CDASH=false \
-    --env WITH_QT5=true \
-    --env BUILD_WITH_QT6=false \
+    --env WITH_QT5=false \
+    --env BUILD_WITH_QT6=true \
```

Comment:
```suggestion
```

Why keeping those? BUILD_WITH_QT6 is now defaulted to TRUE, no need to force it. And can we not get rid of WITH_QT5 and removing [this](https://github.com/qgis/QGIS/blob/master/.docker/docker-qgis-build.sh#L115)